### DISCUSSION TITLE
'useRouter' is not exported

### DISCUSSION THREAD

**Discussion by anoop-gupt** [vercel/next.js#19741] [Help]
I am using Next Router but during the build getting following error
"'Error during bundle: Error: 'useRouter' is not exported by node_modules/next/router.js "

Any pointer??

**Comment by justintemps** [comment]
How are you importing it? Like this?

`import { useRouter } from 'next/router'
`

**Reply by anoop-gupt** [reply]
Yeah, 
import { useRouter } from 'next/router';
import { I18nContext } from '@spark/translations';

**Reply by justintemps** [reply]
Which version of Next.js are you using?

**Reply by anoop-gupt** [reply]
"next": "10.0.3"

**Reply by justintemps** [reply]
Can you share a repro repo?

**Reply by anoop-gupt** [reply]
I am afraid, its client-specific but I am using Nrwl NX monorepo and Apollo , here is the code snippet of usage

const CustomApp = ({ Component, pageProps }: AppProps) => {
  const apolloClient = useApollo(pageProps.initialApolloState);
  const { locale } = useRouter();

  return (
    <I18nContext.Consumer>
